test/Microsoft.ML.AutoML.Tests/AutoFitTests.cs

```diff
@@ -53,11 +54,14 @@ public void AutoFitMultiTest()
             Assert.Equal(NumberDataViewType.Single, scoredData.Schema[DefaultColumnNames.PredictedLabel].Type);
         }
 
-        [TensorFlowFact]
+        // [TensorFlowFact]
+        [Theory, VaryingTolerance(3)]
         //Skipping test temporarily. This test will be re-enabled once the cause of failures has been determined
-        [Trait("Category", "SkipInCI")]
-        public void AutoFitImageClassificationTrainTest()
+        public void AutoFitImageClassificationTrainTest(int tolerance)
         {
+            // Replacing TensorFlowFact
```

Comment:
Can we add some log message here and in all instances where we are replacing a prior fact with a separate check in code?